apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-sidebar/index.tsx

```diff
@@ -43,20 +57,24 @@ const Button = forwardRef(
 
 function WpcomBlockEditorNavSidebar() {
 	const { toggleSidebar, setSidebarClosing } = useDispatch( STORE_KEY );
-	const [ isOpen, isClosing, postType, selectedItemId, siteTitle ] = useSelect( ( select ) => {
+	const { isOpen, isClosing, postType, selectedItemId, siteTitle } = useSelect( ( select ) => {
 		const { getPostType, getSite } = select( 'core' ) as unknown as {
 			getPostType: ( postType: string ) => null | { slug: string };
 			getSite: () => null | { title: string };
 		};
 
-		return [
-			select( STORE_KEY ).isSidebarOpened(),
-			select( STORE_KEY ).isSidebarClosing(),
-			getPostType( select( 'core/editor' ).getCurrentPostType() ),
-			select( 'core/editor' ).getCurrentPostId(),
-			getSite()?.title,
-		];
-	} );
+		const blockEditorNavSidebarSelect: WpcomBlockEditorNavSidebarSelectors = select( STORE_KEY );
+
+		return {
+			isOpen: blockEditorNavSidebarSelect.isSidebarOpened(),
+			isClosing: blockEditorNavSidebarSelect.isSidebarClosing(),
+			postType: getPostType(
+				( select( 'core/editor' ) as CoreEditorPlaceholder ).getCurrentPostType()
+			),
+			selectedItemId: ( select( 'core/editor' ) as CoreEditorPlaceholder ).getCurrentPostId(),
+			siteTitle: getSite()?.title,
+		};
+	}, [] );
```

Comment:
Does something like `getSite` need to be in the dependency array? I'm guessing not, but wanted to double check. (Would apply to other selectors which also depend on functions like this)